gtsam/hybrid/HybridValues.h

```diff
@@ -37,12 +37,15 @@ namespace gtsam {
  */
 class GTSAM_EXPORT HybridValues {
  private:
-  // VectorValue stored the continuous components of the HybridValues.
+  /// Continuous multi-dimensional vectors for \class GaussianFactor.
   VectorValues continuous_;
 
-  // DiscreteValue stored the discrete components of the HybridValues.
+  /// Discrete values for \class DiscreteFactor.
   DiscreteValues discrete_;
 
+  /// Continuous, differentiable manifold values for \class NonlinearFactor.
```

Comment:
We should have `HybridVectorValues` and `HybridValues`. This is very unclean.

Reply:
I disagree